src/gui/tray/Window.qml

```diff
@@ -262,6 +294,13 @@ Window {
                                         color: parent.parent.hovered ? Style.lightHover : "transparent"
                                     }
                                 }
+
+                                Accessible.role: Accessible.MenuItem
+                                Accessible.name: text
+                                Accessible.description: qsTr("Button that opens the client settings window")

```

Comment:
Similar here – I’d say if the Accessible.name is "Settings" then that extended description is not necessary.

(As always please correct me if I’m wrong @tyrylu @Adriorjalesvidal)